source/blog/2022-01-22-sunsetting-dbal-2.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Sunsetting Doctrine DBAL 2
+authorName: Sergei Morozov
+authorEmail: morozov@tut.by
+permalink: /2022/01/22/sunsetting-dbal-2.html
+---
+
+Since the release of [Doctrine DBAL 3.0.0 ](https://github.com/doctrine/dbal/releases/tag/3.0.0) in November 2020,
+the 2.x release series effectively went into the maintenance mode. In the past year, we've been accepting mostly
+the following types of patches for DBAL 2:
+
+1. Development dependency updates
+2. Security fixes
+3. Improvements to compatibility with PHP 8.1
+4. Improvements in the upgrade path to DBAL 3
+
+Except for dependency updates, at the moment, there are no known issues DBAL 2 that would fall into any of
```

Comment:
This sentence seems to be missing a word

Reply:
Do you mean "issues _in_ DBAL 2"? Or something else is missing?

Reply:
Yes, that's what I mean, I was just not sure what word was missing where.